src/client/__init__.py

```diff
@@ -1634,16 +1634,17 @@ def update_pool(self, pool_id: str, pool: Pool) -> Pool:
 
     @add_headers('client')
     def archive_training(self, training_id: str) -> Training:
-        """Sends training to archive
+        """Archives a training pool.
```

Comment:
I would rather not use collocation "training pool", as there might be confusion between `Training` and legacy pool of training type

Reply:
P.S. if you agree with me on this please replace "training pool" collocation in the other changes as well

Reply:
OK. Changed to just "training".